### ISSUE TITLE
Is there a skiovox exploit for v126?

### ISSUE THREAD

**Issue by Balistic123** [bypassiwastaken/skiovox-helper#140]
Is there a skiovox exploit for v126? if not can someone give me the instructions for the v124 exploit of skiovox?

**Comment by Gaboom63** [comment]
> Is there a skiovox exploit for v126? if not can someone give me the instructions for the v124 exploit of skiovox?

I mean not that's been found but like if you just start trying different things similar to the method theirs a chance you could come across one

**Comment by choice21hax** [comment]
Skiovox v124 re-mastered
REQUIREMENTS - 
Needs to be v124 or higher

1. Sign out 
2. Turn off your internet 
3. Click add person 
4. Hold the esc key 
5. Click apps 
6. Click on any kiosk app 
7. Click add network 
8. Put random letters in and click enter 
9. Click diagnose 
10. Click wifi
11. Click open in settings 
12. Close out the diagnose widow (click the X at the top right)
13. Click sign in as existing user (it *should* show up now)
14. Sign in 
15. Close out the school window (or double click the top and click the window behind it)
16. Profit. If your school policy blocked google, use a proxy like interstellar to browse the internet.

**Comment by Balistic123** [comment]
> Skiovox v124 re-mastered REQUIREMENTS - Needs to be v124 or higher
> 
> 1. Sign out
> 2. Turn off your internet
> 3. Click add person
> 4. Hold the esc key
> 5. Click apps
> 6. Click on any kiosk app
> 7. Click add network
> 8. Put random letters in and click enter
> 9. Click diagnose
> 10. Click wifi
> 11. Click open in settings
> 12. Close out the diagnose widow (click the X at the top right)
> 13. Click sign in as existing user (it _should_ show up now)
> 14. Sign in
> 15. Close out the school window (or double click the top and click the window behind it)
> 16. Profit. If your school policy blocked google, use a proxy like interstellar to browse the internet.

@choice21hax when I click add person then hold the escape key it just closes out of the add person, could you send a video tutorial?

**Comment by Balistic123** [comment]
@choice21hax can you provide a video tutorial?

**Comment by Balistic123** [comment]
@choice21hax

**Comment by Gaboom63** [comment]
> Skiovox v124 re-mastered REQUIREMENTS - Needs to be v124 or higher
> 
> 1. Sign out
> 2. Turn off your internet
> 3. Click add person
> 4. Hold the esc key
> 5. Click apps
> 6. Click on any kiosk app
> 7. Click add network
> 8. Put random letters in and click enter
> 9. Click diagnose
> 10. Click wifi
> 11. Click open in settings
> 12. Close out the diagnose widow (click the X at the top right)
> 13. Click sign in as existing user (it _should_ show up now)
> 14. Sign in
> 15. Close out the school window (or double click the top and click the window behind it)
> 16. Profit. If your school policy blocked google, use a proxy like interstellar to browse the internet.

Alright my internet will like auto connect to the schools after step 8 when it fails, what should I do? Also I dont have the add user button

**Comment by JackWagon885** [comment]
yes there is, https://the-wagonization.github.io/The-Wagon-Site/Pages/Bypasser'sChromekit.html#SKIOVOX125-6Method
